### ISSUE TITLE
Transaction history (blockchain-link)

### ISSUE THREAD

**Issue by jerechinsky** [trezor/trezor-suite#3525]
There are multiple workarounds/patches across the suite regarding targets/amounts/fiat.

They should all be fixed on lower level (blockchain-link) and DB level.

Currently blocked by missing values (like vsize) from blockbook.

-- @szymonlesisz

**Comment by slowbackspace** [comment]
I think there is already an issue for this https://github.com/trezor/trezor-suite/issues/3327

**Comment by jerechinsky** [comment]
duplicate of [#3327](https://github.com/trezor/trezor-suite/issues/3327)
